### DISCUSSION TITLE
Larges get slower and slower

### DISCUSSION THREAD

**Discussion by jbellanca** [mifi/lossless-cut#970] [Support]
I'm using Lossless Cut to split up large ProRes 422 files (one's 600GB, another's 150GB) into smaller segments. It starts our fast on the early segments, but as it gets further in, each segment takes longer and longer and longer. Does anyone have a fix for that so that clips at the end don't take hours to export when ones at the start of the file take only seconds?

**Comment by mifi** [comment]
Im suspecting that seeking is taking longer the further into the file the seek happens. In the help screen you can see the cut commands being logged. Maybe it will be fixed in future versions of ffmpeg. If you want you can try to run ffmpeg -ss TIME -i prores-file.mov -c copy out.mov and see if it takes longer and longer the higher ss value you're using (ss is in seconds)
